tasks/php_install.yml

```diff
@@ -1,4 +1,23 @@
 ---
+- name: "[INSTALL] - Debian 11 only - repository for php 8.0"
+  block:
+    - name: Debian 11 - php8 repo - install required packages
+      ansible.builtin.package:
+        name: gpg
+        state: present
+    - name: Debian 11 - php8 repo - GPG key
+      ansible.builtin.apt_key:
+        url: https://packages.sury.org/php/apt.gpg
+        state: present
+    - name: Debian 11 - php8 repo - APT source
```

Comment:
```suggestion
    - name: [Install php] - Add php8 apt repo
```

Reply:
I've proposed new naming, with readable purpose of step. To indicate that we are adding support for custom repo.